src/bin/vip-wp.js

```diff
@@ -368,6 +370,12 @@ commandWrapper( {
 
 		let countSIGINT = 0;
 
+		if ( ! opts.env.isK8sResident ) {
```

Comment:
Should we do an explicit check that it's a WP Cloud site?

Reply:
Great idea!

Done: https://github.com/Automattic/vip-cli/pull/2327/commits/3ab374e0e888348265b7f96d1a0807d9974e76dc#diff-eaa8488f5301b9eb6525750493ddf8fdfbe68a5e1fe7da1866156637722920ddR374